docs/multiplayer.md

```diff
@@ -1,225 +1,208 @@
-     server (if desired). Otherwise you'll have to tell players your IP.
-    - port 3977 if use of the admin interface is desired (see admin_network.txt)
- - Click "multiplayer" on the startup screen
- - Click "start server"
- - Type in a game name
- - Select the type of game ('LAN/Internet' or 'Internet (advertise)'. With the
-   last one other people are able to see you online. Else they need your IP and
-   port to join)
- - Click "start game", "load game" or "load scenario"
- - Start playing
-
+- Click on "Multiplayer" in the Start Menu.
+- Click on "Start Server".
+- Give your server a name.
+- Select the visibility of your server:
+  - "Public": your server will be publicly listed.
+  - "Invite Only": only players who have the invite code for your server can join.
+  - "Local": only players on your local network can join.
+- (optional) Set a password for your server.
+- Click "New Game", "Load Game", or "Play Scenario".
+- Start playing.
 
 ## 2.0) Connecting to a server
 
- - Click "multiplayer" on the startup screen
- - If you want to connect to any network game in your LAN click on 'LAN', then
-   on 'Find Server'
- - If you want to see which servers all online on the Internet, click on
-   'Internet' and 'Find Server'
- - If there were more than one server
-    - select one in the list below the buttons
-    - click on 'join game'
- - If you want to play and you have the ip or hostname of the game server you
-   want connect to.
-    - click add server
-    - type in the ip address or hostname
-    - if you want to add a port use :<port>
- - If you want to play and you have the invite code of the game server you
-   want connect to.
-    - click add server
-    - type in the invite code
- - Now you can select a company and press: "Join company", to help that company
-    - Or you can press "Spectate game", to spectate the game
-    - Or you can press "New company", and start your own company (if there are
-   slots free)
- - You see a progressbar how far you are with joining the server.
- - Happy playing
+- Click on "Multiplayer" in the Start Menu.
+- There are three ways to join a server:
+  - If you want to connect to a local server, click "Search LAN".
+  - If you want to connect to a public game, click "Search internet".
+  - If the server-owner shared an invite code with you:
+    - Click "Add Server".
+    - Fill in the invite code, which always starts with a +.
+    - Click "OK".
+- Click on the server you want to join.
+- Click "Join Game".
+- If the server has a password, it will ask you for this.
+- You see a progressbar how far you are with joining the server.
+- Happy playing.
 
 ## 2.1) Connecting to a server over the console
 
- - Open the console and type in the following command:
-    connect `<ip/host>:<port>#<company-no>`
-
+- Open the console and type in either of the following commands:
+  - `connect <ip/host>:<port>#<company-no>`
+  - `connect <invite-code>#<company-no>`
```

Comment:
Both the port and company number are optional, though this seems to imply they are required. I'm not sure whether it is worth it complicating this document, or just refer to the connect function in the console.